examples/tutorials/01_basics/05_motors.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2016 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Drive Cozmo's wheels, lift and head motors directly
+
+This is an example of how you can also have low-level control of Cozmo's motors
+(wheels, lift and head) for fine-grained control and ease of controlling
+multiple things at once.
+'''
+
+import time
+
+import cozmo
+
+
+def cozmo_program(robot: cozmo.robot.Robot):
+    # Tell the head motor to start lowering the head (at 5 radians per second)
+    robot.move_head(-5)
```

Comment:
tangentially related - would these low level operations be easier to work with if they also accepted an Angle type?

These examples could then read `robot.move_head(degrees(180))` .. though then i'm going to wish the method was called `robot.set_head_move_speed` or somesuch :-/